src/igvfd/types/sample.py

```diff
@@ -90,7 +90,10 @@ class Biosample(Sample):
     embedded_with_frame = Sample.embedded_with_frame + [
         Path('sample_terms', include=['@id', 'term_name']),
         Path('disease_terms', include=['@id', 'term_name']),
-        Path('treatments', include=['@id', 'treatment_term_name', 'purpose']),
+        Path('treatments', include=['@id', 'treatment_term_name', 'treatment_type',
+             'purpose', 'amount', 'amount_units', 'duration', 'duration_units']),
+        Path('sorted_fraction', include=['@id', 'accession']),
+        Path('modifications', include=['@id', 'modality', 'cas', 'cas species', 'fused_domain', 'tagged_protein'])
```

Comment:
cas species should have an underscore: "cas_species"

Reply:
done